src/gui/torrentcontentmodel.cpp

```diff
@@ -113,31 +106,23 @@ namespace
         {
             const QString ext = info.suffix();
             if (!ext.isEmpty()) {
-                const QPixmap pixmap = pixmapForExtension(ext, QSize(32, 32));
-                if (!pixmap.isNull())
-                    return QIcon(pixmap);
+                auto cacheIter = m_iconCache.find(ext);
```

Comment:
I'm not in a position to test in Windows right now so I'd prefer to leave it for a separate PR.